.travis.yml

```diff
@@ -15,10 +14,16 @@ before_script:
   - composer update --prefer-source --ignore-platform-reqs
 
 script:
+  - ./vendor/bin/phpstan analyse
   - ./vendor/bin/phpunit $PHPUNIT_FLAGS
   - php examples/index.php > /dev/null
-  - if php -i |grep -qE xdebug; then ./vendor/bin/humbug; fi
+  - if php -i |grep -qE xdebug; then ./vendor/bin/infection; fi
```

Comment:
As far as i can see, there is no global state or dependency on a db/file system. So we could add `--threads=4` to speed up infection.

Reply:
Also adding `--log-verbosity=none` will make it so infection doesn't write a log. Since it only costs time, and doesn't provide a benefit on ci.

Reply:
:+1: shall apply